style.css

```diff
@@ -0,0 +1,457 @@
+/* Styles to cascade down */
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  color: #333;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  line-height: 20px;
+}
+
+img {
+  max-width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+figcaption {
+  font-size: 15px;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  line-height: 20px;
+  color: #666;
+}
+
+/* https://stackoverflow.com/questions/49701408/css-grid-auto-height-rows-sizing-to-content/49701488 */
+
+body {
+  display: grid;
+  grid-template-rows: minmax(min-content, max-content);
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+}
+
+nav#fixed-top-nav {
+  background-color: #fff;
+  z-index: 2;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 0.1em;
+  display: grid;
+  grid-template-rows: minmax(min-content, max-content);
+  justify-content: center;
+  align-items: stretch;
+  grid-template-columns: 20% 65% 15%;
+  border-bottom: 1px solid silver;
+}
+
+/* nav bar icon svg sizes */
+#menu-hamburger-icon {
+  width: 1em;
+  height: 1em;
+  margin: 0 1em;
+}
+
+#search {
+  width: 1em;
+}
+
+#new-york-times-heading {
+  width: 14em;
+  height: 2em;
+}
+
+/* Aligning nav bar within items */
+.nav-first {
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-columns: 1fr 1fr 3fr;
+  align-items: center;
+  justify-content: center;
+}
+
+#space-cosmos {
+  font-weight: 900;
+  line-height: 1em;
+  font-size: 0.9em;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+}
+
+.nav-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
You are using all these stylings many times differently, you can make a new class named `.flex-center` and use it.